### DISCUSSION TITLE
No access

### DISCUSSION THREAD

**Discussion by pjsm41390** [community/community#13981] [Copilot Conversations]
Hi, I got an email saying I have access but when I try and use it, it says I don't on VS code and Visual studio.

**Comment by davecheney** [comment]
There is no waitlist entry for @[pjsm41390](https://github.com/pjsm41390), did you perhaps register under a different GitHub account?

**Reply by officialzachward** [reply]
@davecheney I have the same problem. I got an email today, and vscode says I have no access.

**Reply by davecheney** [reply]
I've checked that your invite is active. Can you describe the problem you are having please.

**Reply by officialzachward** [reply]
@davecheney Well after more digging it seems it's an issue specifically with Visual Studio Code. I now have it working on Visual Studio 2022. But yeah the docs aren't particularly helpful with Visual Studio Code. I even triggered the authentication and logged in with my Github account (so I think), but visual studio code still fails to see that I have access to Github Copilot. Whenever I try to enable it, it says please join the waitlist :)

**Reply by davecheney** [reply]
@officialzachward are you using VSCode, or one of the source recompiled derivatives like OSS Code or VSCodium?

**Comment by JustTrynaBust** [comment]
I got invited to it but still says i dont have access and i am using the right account

**Reply by davecheney** [reply]
@JustTrynaBust I can see that you have been sent an invite.

**Reply by officialzachward** [reply]
@davecheney Well...do you have any advice? Any help would be appreciated. It's quite the tease to get "access" and not actually have it.
